platforms/karaf/features/src/main/resources/feature.xml

```diff
@@ -25,6 +25,8 @@
     <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson.bundle.version}</bundle>
     <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/${jackson.bundle.version}</bundle>
     <bundle dependency='true'>mvn:org.yaml/snakeyaml/${snakeyaml.version}</bundle>
+    <bundle dependency='true'>mvn:org.bouncycastle/bcprov-ext-jdk15on/${bouncycastle.version}</bundle>
```

Comment:
Karaf needs to bundle this dependency? I thought they were optional

Reply:
oops, I forgot to update

Reply:
strange, karaf tests started breaking when I removed them :confused: